lib/index-static.jsx

```diff
@@ -39,8 +39,8 @@ function generateWithPageHTML(url, options, pageHTML) {
           return <meta key={i} name={name} content={options.meta[name]}/>;
         })}
         <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,700,600italic,700italic,800,800italic"/>
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"/>
```

Comment:
Hmm, while I don't mind using a CDN for produciton, i'd prefer we use that so we can easily develop locally if needed. (Our use of Open Sans should also be local but I never got around to doing that... and also, the site doesn't look much different using open sans vs. Arial anyhow...)